src/docfx/publish/PublishItem.cs

```diff
@@ -2,15 +2,26 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Serialization;
 
 namespace Microsoft.Docs.Build
 {
+    [JsonObject(NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
```

Comment:
can we add it to default setting?

Reply:
I hope so, that i'll be a separate PR once #3846 is in